var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -19,6 +19,7 @@ class Raja(CachedCMakePackage, CudaPackage, ROCmPackage):
 
     version('develop', branch='develop', submodules='True')
     version('main',  branch='main',  submodules='True')
+    version('2022.03.0', tag='v2022.03.0', submodules=True)
```

Comment:
Do we use any of the submodules in the package build? If not, perhaps we should turn them off so they can't be used by mistake?